Simulator/augments.py

```diff
@@ -0,0 +1,52 @@
+from enum import Enum, auto
+
+
+class Augment(Enum):
+    AFK = auto()
+
+    @staticmethod
+    def is_augment_round(round_:int) -> bool:
+        return round_ in [3,10,17]
+
+
+class SelectedAugmentConfig:
+    round: int
+    augment: Augment
+
+    def __init__(self, augment: Augment, round_: int):
+        self.round = round_
+        self.augment = augment
+
+
+def handle_start_round_augment(player, augment_config: SelectedAugmentConfig):
+    """
+    Handle modifications to the player at the start of the round
+    """
+    if augment_config.augment == Augment.AFK:
```

Comment:
If using truncations, this if statement isn't needed.
Also, instead of using a bunch of if statements here to check individual augments. Create a list in the stats that has all of the state round augments or all of the state of battle augments then check if the augment we are looking at is included in that list. If using if statements, you are looking O(N) time on checks for however many augments you have + the space and load time required to load the variable to check each time. If you use a list that is sorted, the check is O(log(n)) and the load time requirement is only once because it will put the whole list in memory. It's even better because it will load a block of memory at a time and our lists will likely fit inside one block.